continuous-integration/deployment/check_expected_artifacts.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+
+
+def _get_expected_artifacts():
```

Comment:
I think we can drop the `_get` prefix here since we are just reading a file plus some more

Reply:
Sure. Same for the other function too then.